docs/types/value-types.rst

```diff
@@ -310,10 +312,12 @@ Every :ref:`contract<contracts>` defines its own type.
 You can implicitly convert contracts to contracts they inherit from.
 Contracts can be explicitly converted to and from the ``address`` type.
 
-Explicit conversion to and from the ``address payable`` type
-is only possible if the contract type has a payable fallback function.
-The conversion is still performed using ``address(x)`` and not
-using ``address payable(x)``. You can find more information in the section about
+Explicit conversion to and from the ``address payable`` type is only possible
+if the contract type has a payable fallback function.  The conversion is still
+performed using ``address(x)``. If the contract type does not have a payale
```

Comment:
```suggestion
performed using ``address(x)``. If the contract type does not have a payable
```

Reply:
Don't we actually have a spell checker to find something like this?

Reply:
Yea, weird.
Fixed typo